not sure why hound is not running? but running lint shows:
```
webpack/simple_named_modules.js
  10:1   error  Expected newline before "use strict" directive         lines-around-directive
  10:1   error  'use strict' is unnecessary inside of modules          strict
  10:1   error  Strings must use singlequote                           quotes
  18:21  error  Strings must use singlequote                           quotes
  19:26  error  Strings must use singlequote                           quotes
  21:11  error  Expected space(s) after "if"                           keyword-spacing
  22:13  error  Assignment to property of function parameter 'module'  no-param-reassign
  23:72  error  Missing trailing comma                                 comma-dangle
  25:13  error  Assignment to property of function parameter 'module'  no-param-reassign
  25:80  error  Missing semicolon                                      semi
```

--- a/.eslintignore
+++ b/.eslintignore
@@ -2,3 +2,4 @@
 app/**
 vendor/**
 config/webpack.config.js
+webpack/simple_named_modules.js

--- a/config/webpack.config.js
+++ b/config/webpack.config.js
@@ -10,8 +10,7 @@ var CompressionPlugin = require('compression-webpack-plugin');
 var LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
 var pluginUtils = require('../script/plugin_webpack_directories');
 var vendorEntry = require('./webpack.vendor');
-
-
+var SimpleNamedModulesPlugin = require('../webpack/simple_named_modules');
 
 module.exports = env => {
   // must match config.webpack.dev_server.port
@@ -163,7 +162,7 @@ module.exports = env => {
         },
         sourceMap: false
       }),
-      new webpack.NamedModulesPlugin(),
+      new SimpleNamedModulesPlugin(),
       new webpack.optimize.ModuleConcatenationPlugin(),
       new webpack.optimize.OccurrenceOrderPlugin(),
       new CompressionPlugin()

--- a/webpack/simple_named_modules.js
+++ b/webpack/simple_named_modules.js
@@ -1,0 +1,35 @@
+/*
+  Simple Named Modules Plugin
+  Strips relative path up to node_modules/ from the module ID.
+  This allows for consistent module IDs when building webpack bundles from
+  differing base paths relative to the node_modules directory.
+
+  Based on NamedModulesPlugin by Tobias Koppers @sokra, originally licensed under
+  MIT License: http://www.opensource.org/licenses/mit-license.php
+*/
+"use strict";
+
+class SimpleNamedModulesPlugin {
+  constructor(options) {
+    this.options = options || {};
+  }
+
+  apply(compiler) {
+    compiler.plugin("compilation", (compilation) => {
+      compilation.plugin("before-module-ids", (modules) => {
+        modules.forEach((module) => {
+          if(module.id === null && module.libIdent) {
+            module.id = module.libIdent({
+              context: this.options.context || compiler.options.context
+            });
+            if (module.id.includes('node_modules')) {
+              module.id = module.id.slice(module.id.indexOf('node_modules'))
+            }
+          }
+        });
+      });
+    });
+  }
+}
+
+module.exports = SimpleNamedModulesPlugin;

